utils/incrparse/test_util.py

```diff
@@ -25,7 +25,7 @@ def run_command(cmd):
     if sys.version_info[0] < 3:
         cmd = list(map(lambda s: s.encode('utf-8'), cmd))
     print(' '.join([escapeCmdArg(arg) for arg in cmd]))
-    return subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+    return subprocess.check_output(cmd, encoding='utf-8', stderr=subprocess.STDOUT)
```

Comment:
Does Python 2.7 support the encoding argument?  I don't see it at 

https://docs.python.org/2.7/library/subprocess.html?highlight=subprocess#subprocess.check_output

Reply:
Bleh, I just read the docs more carefully - it was introduced in Python 3.6, which makes it just a bit too late for us.